expense_app/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
```

Comment:
Why commented out?

Reply:
same as admin.py , failed flake8 for not using the import